updated readme for putt-putt-miss

SarahMalykke opened the pull request:
New update 🙂 @jessb0t

Commit:
updated readme for putt-putt-miss

jessb0t commented:
@SarahMalykke Please take a look at the changes we made to the dataset readme as some will apply here.  Additionally:
- please replace the Roadmap section with your planned output products (e.g. paper, poster, etc.).  I desperately need to update mine, but [here is some inspiration](https://github.com/NDCLab/readAloud-valence-alpha/tree/dev).
- please update contributors.  Since this is the analysis project, you should only include those who you anticipate will be co-authors on your products at this time.  (Can always change down the road, of course, but I wouldn't list individuals who are merely helping with data collection here.)
Thanks!

jessb0t committed:
Add https

jessb0t committed:
Fix typo

SarahMalykke committed:
Update README.md

Updated the contributors to only include potential authors (for now). The roadmap will definitely be updated once I discuss more with George @jessb0t

SarahMalykke commented:
Updated directly on GitHub since it was minor edits @jessb0t

jessb0t commented:
Thanks, @SarahMalykke !  I think the readme needs some tweaks, too, to mirror the changes we made over on putt-putt-dataset?  (For instance, present tense.)

SarahMalykke commented:
Ah, since this is the analysis repo, should I write in past tense, as if data collection is complete? 🙂 @jessb0t

jessb0t commented:
@SarahMalykke I would actually keep present tense with a focus on the analysis (not data collection).  Imagine a researcher coming in two years to check it out...the facts of the analysis would be in the present for them (even if they are in the past for you). 😉

SarahMalykke committed:
Update README.md

I think these changes are much better 😅 @jessb0t I now see the difference between putt-putt-dataset and putt-putt-miss 🙂

SarahMalykke commented:
Updated directly on the doc 🙂 @jessb0t

jessb0t committed:
Correct typo

jessb0t commented:
@SarahMalykke Lookin' good! 😎 Merged!
